src/test/java/org/folio/rest/api/InstanceStorageTest.java

```diff
@@ -1625,6 +1627,78 @@ public void shouldNotChangeStatusUpdatedDateWhenStatusHasNotChanged() throws Exc
       .getString(STATUS_UPDATED_DATE_PROPERTY), withinSecondsBeforeNow(seconds(2)));
   }
 
+  private void assertExists(JsonObject expectedInstance) throws Exception {
```

Comment:
Given that this method is used by multiple tests, I think it is hard to place it near any one usage, and would be better moved to after the tests

Reply:
Yes, I've reordered the methods.